docker-compose.yml

```diff
@@ -19,7 +19,7 @@ test:
   volumes:
     - .:/app
   working_dir: /app
-  command: node_modules/mocha/bin/mocha --harmony test/setup.js test/**/*.js
+  command: node_modules/mocha/bin/mocha --harmony test/setup.js test/**/*.js ./lib/**/*.spec.js
```

Comment:
We encountered some troubles with globs. I would recommend using following syntax instead:

```
NODE_ENV=test ./node_modules/.bin/mocha \
		./tests/setup.js \
		$$(find tests/e2e -name '*.spec.js')
```